salt/pillar/journal.sls

```diff
@@ -52,6 +52,9 @@ journal:
 
     cache_control: private, no-cache, no-store, must-revalidate
 
+    rds_articles:
+        16846: https://repro.elifesciences.org/example.html
```

Comment:
This is only used in local tests; it is not the id of the real article that will use this feature, but I took the same one contained in `api-dummy` that is used now in the `journal` test.